src/osm_configurator/view/popups/yes_no_pop_up.py

```diff
@@ -27,37 +37,73 @@ def __init__(self, message: str, func):
             message (str): The message to be shown in the PopUp.
             func (typing.Callable): Function taking one Boolean and has no return for the PopUp to return a message
         """
-        self.func = func
-
-        popup = super().__init__()
-        self.geometry(POPUPSIZE)
-
-        self.title("Alert")
-
-        customtkinter.CTkLabel(popup, text=message) \
-            .pack(side="top", fill="both", expand="True", padx=10, pady=10)
-
-        customtkinter.CTkButton(self, text="Accept", command=combine_funcs(self.accept, self.destroy)) \
-            .pack(side="bottom", padx=10, pady=10)
-
-        self.button = customtkinter.CTkButton(self, text="Cancel",
-                                              command=combine_funcs(self.cancel, self.destroy)) \
-            .pack(side="bottom", padx=10, pady=10)
 
-    def accept(self):
+        self._func = func
+
+        super().__init__(
+            fg_color=frame_constants_i.FrameConstants.HEAD_FRAME_FG_COLOR.value,
+        )
+
+        # Configuring the grid
+        self.grid_columnconfigure(0, weight=1)  # Topbar: showing nothing
```

Comment:
seeing you commedn you might have swaped around column and row, row is for up and down and column is left and right